app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Well done adding the devise to your project. However, there is one requirement with devise that is missing with is to ensure that the user should confirm their email before being able to sign up on the app.

### Requirement :point_down: 
![Screenshot from 2023-09-07 14-50-55](https://github.com/binodbhusal/BlogApp/assets/89228743/c5a42fe2-fbf1-44ca-a912-a11f30bf35f4)


So to achieve that requirement you should enable the `:confirmable` feature. Please do as follow :point_down: 
```suggestion
  # Include default devise modules. Others available are:
  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
  devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :confirmable
```